sonar-project.properties

```diff
@@ -0,0 +1,30 @@
+# Project identification
+sonar.projectKey=nodejs-tracer
+sonar.projectName=NodeJS Tracer
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarqube.instana.io
+sonar.token=${{ secrets.SONAR_TOKEN }}
+
+# Source code locations
+sonar.sources=packages
+sonar.scm.disabled=true
+
+# Encoding of the source code. Default is default system encoding
+sonar.sourceEncoding=UTF-8
+
+# Language and file suffixes
+sonar.language=js
+sonar.c.file.suffixes=-
+sonar.cpp.file.suffixes=-
+sonar.objc.file.suffixes=-
+
+# Code coverage configuration
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+# ESLint configuration
+sonar.eslint.reportPaths=eslint-report.json
+
+# Code analysis parameters  
+sonar.inclusions=**/src/**,**/lib/**
+sonar.exclusions=**/test/**,**/dummy-app/**,**/precompile/**.,**/uncaught/**
+sonar.coverage.exclusions=**/test/**,**/dummy-app/**,**/precompile/**,**/uncaught/**
```

Comment:
I understand all exclusions except for the `**/uncaught/**` one. That would exclude `packages/collector/src/uncaught` from coverage analysis as well as from normal static code analysis, right? Why would we want to exclude that?

Reply:
The code coverage for the file `syncHttp.js` in this particular folder was 0, that's why I excluded this from the analysis, Removing it from here will not have a significant impact on the overall coverage.

Reply:
I would suggest not excluding it, since it is part of our production code. Excluding it feels a bit like cheating. If it is uncovered, it is uncovered, so be it.

Side note: For some reason `syncHttp.js` does not show up at all in our HTML coverage report https://output.circle-artifacts.com/output/job/dd17c749-4aef-4335-a1c3-497ed8f98fc3/artifacts/0/coverage/lcov-report/collector/src/uncaught/index.html, I wonder why... 

Oh, wait! I think `syncHttp.js` file is completely unused and we can actually delete it! That's why it also has 0% test coverage :D 

Looks like we removed its usage here: https://github.com/instana/nodejs/commit/fb6570a862dbdec776eb78b840dcdc4184cd5f00#diff-8ff40b8dde5d139718e5cd429da1d02c4aac512218fa125e3947c0b0e049df8cL481 but forgot to delete the file itself. We can delete that file in a different PR or in this PR, whatever you prefer.

Cool, so Sonar already uncovered an actual code quality issue. Yay, Sonar!

Okay, long story short, let's remove `**/uncaught/**` from both `sonar.exclusions` and `sonar.coverage.exclusions`.